package.py

```diff
@@ -1,6 +1,6 @@
-name = ""
-title = ""
-version = "0.0.0"
+name = "ayon-addon-template"
+title = "Ayon-Addon-Template"
+version = "0.1.0"
```

Comment:
I would keep values empty, but add comments
```suggestion
# Required: Addon name e.g. 'deadline'
name = ""
# Optional: Addon title shown in UI, 'name' is used by default e.g. 'Deadline'
title = ""
# Valid semantic version (https://semver.org/spec/v1.0.0.html)
version = "0.0.0"
```

Reply:
With `name` empty - the example addon doesn't actually "work" out of the box? Which may be unwanted?

Reply:
I would say it is actually wanted... It's like: Hey, you have to change this.

Reply:
That's a good question. So far the tempalte seems to be not valid out-of-the-box but was that intended?

Reply:
> I would say it is actually wanted... It's like: Hey, you have to change this.

Interesting because to me it's a bit the opposite - it would technically be totally fine for an example addon to be uploaded to the server for someone to 'test' with? Similar to the example settings addon that @martastain uses (which I believe is a different one). You want to just be able to throw that into the server and "test with". In a way the example addon could even be used as a testing platform for the "core" functionality of addons.

Reply:
> Interesting because to me it's a bit the opposite - it would technically be totally fine for an example addon to be uploaded to the server for someone to 'test' with? 

But this is not example, this is template.

Reply:
Updated PR and added comments instead of changing the defaults. Just as @iLLiCiTiT suggested